test/e2e/scenarios/Cluster/cluster_update_with_tags.yaml

```diff
@@ -21,7 +21,7 @@ steps:
         conditions:
           ACK.ResourceSynced:
             status: "True"
-            timeout: 7200
+            timeout: 3600
```

Comment:
It takes an hour to start a memorydb cluster?

Reply:
Same case here. Cluster creation takes less than 30 minutes. If the time is within Snapshot Window, it is going to snapshot cluster. Snapshotting should be done within 30 minutes as well. Hence, I set 1 hour in total to handle this case.

Reply:
Same as above.

Reply:
https://github.com/aws-controllers-k8s/memorydb-controller/pull/57#discussion_r1125143899